api/helpers/game-states/moves/resolve/execute.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
     if (fizzles) {
       result.moveType = MoveType.FIZZLE;
       result.scrap.push(result.oneOff);
+      result.turn++;
```

Comment:
This was a critical bug and I'm surprised to just be catching it now. The turn wasn't incremented when the one-off fizzled.